### ISSUE TITLE
ICC guards and Vanguard have to low HP

### ISSUE THREAD

**Issue by TarasqueAO** [project-rubika/issues#722]
**Describe the bug**
ICC guars and Vanguard Watchers / Commanders have about 227-272 HP on PRK
To compare a lvl 120 Male Senior Watcher on live servers have 9201 HP.

**To Reproduce**
1. Find a guard
2. Click on Information Tool - Life and Nano Points
3. Verify HP

**Expected behavior**
HP on several guard types should be increased.

**Screenshots**
![image](https://github.com/project-rubika/issues/assets/166374747/cf348d4a-f045-422d-815f-b41e5ae7bd5f)
![image](https://github.com/project-rubika/issues/assets/166374747/45a083e5-30ca-47be-bebc-e328ca6547ac)

**Playfield info:**

```
Extended location information:
- 382.6, 481.4, 8.0 (382.6 481.4 y 8.0 540)
- Pf Proxy: Model=51100:540 GS=0 SG=0 R=540, resource: 540
- zone: 249, area: "Old Athen West"
Server id: 1, character id: 50000:1479, time: 2024-04-11 09:10:36 (UTC)
Version: 00.6.1_EP1, map id: NOT FOUND, build: 52606.

```
----

**Comment by TarasqueAO** [comment]
From Avalon Omni Bases:
Omni-Pol Command Juggernaut [247 Soldier]      life 	1 901 860
General Freewheeler [250 Soldier]              life 	1 200 000
Supply Master Eel [300 Agent]                  life 	4 200 000
General Kaehler Jr. [300 Soldier]              life 	2 800 000

From Tir:
Urban Defense Patroller [200 Soldier]    life 	43720
Urban Guard Patroller [200 Enforcer]     life 	43720
Sentinel Commander Higgins [225 MartialArtist]    life 	908
Supreme Commander Silverstone [300 Soldier]    life 	4 200 000

**Comment by prkdrake** [comment]
Potentially a duplicate of #697

**Comment by RaggyLP** [comment]
![image](https://github.com/user-attachments/assets/d0f517a6-3bb8-4003-9b66-e6106833ae6c)
![image](https://github.com/user-attachments/assets/20b37575-3c10-43b2-a5d7-56a0d0301db0)
![image](https://github.com/user-attachments/assets/d983dd8d-8749-4e6c-8abf-74e6d85372c3)
![image](https://github.com/user-attachments/assets/4a0f0149-865f-4384-9e5d-ee818d824246)

Spot checks show that this appears to be fixed following the template changes from #697
Closing as fixed as of 25/07/24
